doc/sphinxman/source/adc.rst

```diff
@@ -38,38 +38,276 @@
+are supported with all ADC methods using the adcc backend.
+
+Currently adcc is only capable of performing in-core calculations,
+for which, however, permutational symmetry and spin symmetry is taken
+into account for both tensor computations and tensor storage.
+Inside adcc some heuristic checks for overly excessive memory requirements
+are implemented, resulting in a warning in case a
+successful execution is unlikely. There are no guarantees for the memory
+to be sufficient in case such a warning is not displayed.
+
+More detailed documentation about adcc and its features can be found
+at `<https://adc-connect.org>`_,
+especially the `theory section <https://adc-connect.org/latest/theory.html>`_.
+If you are using adcc from |PSIfour| for your calculations,
+please cite both |PSIfour| as well as adcc [Herbst2019]_
+in your published work.
+
+**The ADC wavefunction object.**
+After running the ADC calculation in adcc, the interface code sets
+the following variables in the returned :py:class:`~psi4.core.Wavefunction`
+in case they have been computed:
+
+* Ground state energy terms like ``MP2 correlation energy``, ``MP3 correlation energy``,
+  ``MP2 total energy``, ``MP3 total energy``, ``current correlation energy`` and ``current energy``.
+* ``number of iterations``: The number of iterations the iterative solver required to converge.
+* ``number of excited states``: The number of excited states, which were computed.
+* ``excitation energies``: The obtained excitation energies as a :py:class:`~psi4.core.Matrix`.
+* ``MP2 dipole X`` and the other components: Ground state dipole moments at MP(2) level.
+* ``transition dipoles``, ``oscillator strengths`` and ``dipoles``:
+  The respective properties as :py:class:`~psi4.core.Matrix`
+
+The following attribute is set on returned wavefunctions:
+
+* ``adcc_state``: The `adcc.ExcitedStates <https://adc-connect.org/q/excitedstates>`_
+  object used by adcc to store the ADC(n) excitation energies and all precomputed data
+  in the format used by adcc.
+  Provides direct access to analysis and plotting capabilities from adcc.
+  For example ``adcc_state.plot_spectrum()`` plots a broadened excited states spectrum
+  in matplotlib. See the `adcc documentation <https://adc-connect.org/latest/calculations.html>`_
+  for details.
+
+**Tips for convergence issues.**
+If you encounter convergence issues inside adcc, the following parameters
+are worth tweaking:
+
+* |adc__max_num_vecs|: Specifies the maximal number of subspace vectors
+  in the Jacobi-Davidson scheme before a restart occurs. The defaults are usually
+  good, but do not be shy to increase this value if you encounter convergence problems.
+* |adc__num_guesses|: By default adcc uses twice as many guess vectors as
+  states to be computed. Sometimes increasing this value by a few vectors can be helpful.
+  If you encounter a convergence to zero eigenvalues, than decreasing this parameter might
+  solve the problems.
+
+.. _`sec:adcbuiltin`:
+
```

Comment:
Many thanks for good docs with both theoretical background and practical tips.

Reply:
Well, not all is my work in this file, so I can't take all the credit ...